pyproject.toml

```
@@ -29,7 +30,7 @@ classifiers = [
 
 [project.optional-dependencies]
 plotting = ["matplotlib"]
-resources = ["pythia8mc", "matplotlib", "pandas"]
+resources = ["pythia8mc", "matplotlib"]
```

Maybe change this to be more explicit now that `pandas` is default.
```suggestion
pythia8 = ["pythia8mc", "matplotlib"]
```